trin-core/src/portalnet/types.rs

```diff
@@ -219,6 +234,17 @@ pub struct FoundContent {
     pub payload: Vec<u8>,
 }
 
+#[derive(Debug, PartialEq, Clone, Encode, Decode)]
+pub struct Offer {
+    pub content_keys: Vec<u8>,
+}
+
+#[derive(Debug, PartialEq, Clone, Encode, Decode)]
+pub struct Accept {
+    pub connection_id: Vec<u8>,
+    pub content_keys: Vec<u8>
```

Comment:
For change to SSZ Bitlist

```suggestion
    pub content_keys: Vec<bool>
```